files/zh-cn/web/api/filesystemsyncaccesshandle/index.md

```diff
@@ -0,0 +1,88 @@
+---
+title: FileSystemSyncAccessHandle
+slug: Web/API/FileSystemSyncAccessHandle
+---
+
+{{securecontext_header}}{{APIRef("File System API")}}
+
+{{domxref("File System API", "文件系统", "", "nocode")}} 的 **`FileSystemSyncAccessHandle`** 接口用于表示一个指向某个文件系统条目的同步句柄。
+
+这个类只在专用于处理对末端用户不可见的[源私有文件系统](/zh-CN/docs/Web/API/File_System_API/Origin_private_file_system)上的文件的 [Web Worker](/zh-CN/docs/Web/API/Web_Workers_API) 中可以访问（这样它的方法就不会阻塞主线程的执行）。
```

Comment:
The [dedicated worker](https://developer.mozilla.org/zh-CN/docs/Web/API/Web_Workers_API/Using_web_workers#%E4%B8%93%E7%94%A8_worker) is a well-known term.

```suggestion
这个类只在用于处理对终端用户不可见的[源私有文件系统](/zh-CN/docs/Web/API/File_System_API/Origin_private_file_system)上的文件的专用 [Web Worker](/zh-CN/docs/Web/API/Web_Workers_API) 中可以访问（这样它的方法就不会阻塞主线程的执行）。
```